examples/step-79/step-79.cc

```diff
@@ -792,9 +792,9 @@ namespace SAND
     // us how far off our filtered density is from the filter
     // applied to the unfiltered density. That is because while at
     // the solution of the nonlinear problem, we have
-    // $\rho=H\sigma$, but at intermediate iterations, we in
-    // general have $\rho^k\neq H\sigma^k$ and the "residual"
-    // $\rho^k-H\sigma^k$ will then appear as the right hand side
+    // $\rho=H\varrho$, but at intermediate iterations, we in
+    // general have $\rho^k\neq H\varrho^k$ and the "residual"
+    // $\rho^k-H\varrho^k$ will then appear as the right hand side
```

Comment:
@juleoc02 Would you agree that this change matches the modifications made in the introduction?

Reply:
Absolutely - good catch!